private/buf/bufgen/config_test.go

```diff
@@ -36,7 +36,7 @@ func TestReadConfigV1Beta1(t *testing.T) {
 			{
 				Name:     "go",
 				Out:      "gen/go",
-				Opt:      "plugins=grpc",
+				Opt:      "plugins=connect",
```

Comment:
This probably doesn't impacts the test itself, but iirc `plugins=grpc` was a way to pass a plugin to `protoc-gen-go` via an option. This functionality was later removed in favor of distinct plugins.

Probably fine because we're testing the shape of the config and not actual behaviour, but might want to get a second opinion whether we want to change this or not for correctness.

Reply:
Makes sense.  @doriable any thoughts?